node-tests/test.js

```diff
@@ -257,4 +258,21 @@ describe('included()', function() {
       expect(cacheKey).to.equal(expectedCacheKey);
     });
   });
+
+  describe('different instances of inline precompile for the same parent', function() {
+    it('should have only 1 inlinePrecompile registered', function() {
```

Comment:
Just to confirm, this test fails before the changes here and passes after?

Reply:
There was no test for non parallel plugin. When I added a test it failed.

Reply:
But overall before this fix the test failed.